src/checksum/power8/crc32.sx

```diff
@@ -775,11 +775,13 @@ FUNC_START(__crc32_vpmsum)
 	b	.Lout
 
 FUNC_END(__crc32_vpmsum)
+#endif
 
 /*
  * Make sure the stack isn't executable with GCC (regardless of platform).
+ * Note: this section must always be output and can never be #ifdef'd out as
+ * part of conditional compilation.
  */
 #ifdef __ELF__
 .section	.note.GNU-stack,"",@progbits
 #endif
```

Comment:
Do you want to add a scary
/* DO NOT SAY #ifndef BELOW THIS LINE, per above comment */

The comment you have is good, but will anyone read it before slapping an #endif on the last line of the file?

Reply:
Will do.